### ISSUE TITLE
Add "comparison" with OpenTitan

### ISSUE THREAD

**Issue by dumblob** [betrusted-io/betrusted-io.github.io#4]
Google [announced](https://opensource.googleblog.com/2019/11/opentitan-open-sourcing-transparent.html ) their hardware-based "root of trust" for any hardware (wearables, mobile, desktops, servers, ...) called [OpenTitan](https://opentitan.org/ ).

OpenTitan seems to get a lot of attention in public, so I thought it would make sense to mention it on betrusted.io web and maybe even try some high-level comparison (if not technical, then comparison of the goals). Otherwise confusion might arise or people (potential betrusted "customers"/users) will simply not understand the benefits of betrusted compared to OpenTitan.

**Comment by bunnie** [comment]
Good point...the main page needs updating anyways, I'll factor this in when it gets a makeover in a month or so...
